Documentation/contributing/development/images.rst

```diff
@@ -198,85 +198,49 @@ Image dependency:
 
-.. image:: ../../images/cilium-quayio-tag-2.png
-    :align: center
+4. Wait for the build to complete. Only after the image is available run:
 
-4. After enabling the trigger, click again on the wheel.
-5. And click on "Run Trigger Now".
+    $ make -C images/ update-runtime-image update-builder-image
 
-.. image:: ../../images/cilium-quayio-tag-3.png
-    :align: center
+5. Commit your changes and re-push to the PR in cilium/cilium.
 
-6. A new pop-up will appear to select your desired branch.
-7. If you're interested in simply bumping the image to have the latest
-   packages, then select the release branch (i.e. v1.7, v1.8). If you already
-   have a branch that contains changes, select the branch that contains the new
-   changes.
+.. code:: bash
 
-.. image:: ../../images/cilium-quayio-tag-4.png
-    :align: center
+    $ git commit --amend -s images/ -m "update cilium-{runtime,builder}"
 
-8. After selecting your branch click on "Start Build".
+6. Ping one of the members of `team/build <https://github.com/orgs/cilium/teams/build/members>`_
+   to approve the build that was created by GitHub Actions `here <https://github.com/cilium/cilium/actions?query=workflow:%22Base+Image+Release+Build%22>`_.
 
-.. image:: ../../images/cilium-quayio-tag-5.png
-    :align: center
+7. Wait for the build to complete. Only after the image is available run:
 
-9. Once the build has started you can disable the Build trigger by clicking on
-   the wheel.
-10. And click on "Disable Trigger".
+    $ make -C images/ update-runtime-image update-builder-image
 
-.. image:: ../../images/cilium-quayio-tag-6.png
-    :align: center
+8. Commit your changes and re-push to the PR in cilium/cilium.
 
-11. Confirm that you want to disable the build trigger.
+.. code:: bash
 
-.. image:: ../../images/cilium-quayio-tag-7.png
-    :align: center
+    $ git commit --amend -s images/ -m "update cilium-{runtime,builder}"
 
-12. Once the build is finished click under Tags (on the left menu).
-13. Click on the wheel and;
-14. Add a new tag to the image that was built.
-
-.. image:: ../../images/cilium-quayio-tag-8.png
-    :align: center
-
-15. Write the name of the tag that you want to give for the newly built image.
-16. Confirm the name is correct and click on "Create Tag".
-
-.. image:: ../../images/cilium-quayio-tag-9.png
-    :align: center
-
-17. After the new tag was created you can delete the other tag, which is the
-    name of your branch. Select the tag name.
-18. Click in Actions.
-19. Click in "Delete Tags".
-
-.. image:: ../../images/cilium-quayio-tag-10.png
-    :align: center
-
-20. Confirm that you want to delete tag with your branch name.
-
-.. image:: ../../images/cilium-quayio-tag-11.png
-    :align: center
+9. Grep and replace all other places where the images are used. (This step will
+   be removed once it is part of the make targets ``update-runtime-image``
+   ``update-builder-image``).
```

Comment:
Maybe mention which terms we should be grepping for? I guess it's `cilium-runtime` and `cilium-builder`?

Reply:
The next bit of text after this step is:

> You have created a new image build with a new tag. The next steps should be to update the repository root's Dockerfile so that it points to the new``cilium-builder`` or ``cilium-runtime`` image recently created.

Does this still make sense in context of the new instructions? I found it confusing whether it's trying to tell me that after I ran the commands above we're in a good spot, or if it's trying to instruct me to take some action.

Reply:
oh I didn't realize there was more after the steps. I've re-wrote the next steps.